Tests/SwiftDocCTests/Semantics/MetadataAvailabilityTests.swift

```diff
@@ -0,0 +1,236 @@
+
+            directive.map { directive in
+                var problems = [Problem]()
+                XCTAssertEqual(Metadata.directiveName, directive.name)
+                let _ = Metadata(from: directive, source: nil, for: bundle, in: context, problems: &problems)
+                XCTAssertEqual(2, problems.count)
+                let diagnosticIdentifiers = Set(problems.map { $0.diagnostic.identifier })
+                XCTAssertEqual(diagnosticIdentifiers, ["org.swift.docc.\(MetadataAvailability.self).DuplicateIntroduced"])
+            }
+        }
+
+        for platform in MetadataAvailability.Platform.allCases {
+            guard platform != .any else { continue }
+
+            let source = """
+            @Metadata {
+                @Available(\(platform.rawValue), introduced: \"1.0\")
+                @Available(\(platform.rawValue), introduced: \"2.0\")
+            }
+            """
+            try assertInvalidDirective(source: source)
+        }
+    }
+
+    func testInvalidDuplicateBeta() throws {
+        throw XCTSkip("FIXME: isBeta is unused (https://github.com/apple/swift-docc/issues/441)")
+
+//        func assertInvalidDirective(source: String) throws {
+//            let document = Document(parsing: source, options: .parseBlockDirectives)
+//            let directive = document.child(at: 0) as? BlockDirective
+//            XCTAssertNotNil(directive)
+//
+//            let (bundle, context) = try testBundleAndContext(named: "AvailabilityBundle")
+//
+//            directive.map { directive in
+//                var problems = [Problem]()
+//                XCTAssertEqual(Metadata.directiveName, directive.name)
+//                let _ = Metadata(from: directive, source: nil, for: bundle, in: context, problems: &problems)
+//                XCTAssertEqual(2, problems.count)
+//                let diagnosticIdentifiers = Set(problems.map { $0.diagnostic.identifier })
+//                XCTAssertEqual(diagnosticIdentifiers, ["org.swift.docc.\(MetadataAvailability.self).DuplicateBeta"])
+//            }
+//        }
+//
+//        for platform in MetadataAvailability.Platform.allCases {
+//            let source = """
+//            @Metadata {
+//                @Available(\(platform.rawValue), isBeta: true)
+//                @Available(\(platform.rawValue), isBeta: true)
+//            }
+//            """
+//            try assertInvalidDirective(source: source)
+//        }
+//
+//        // also test for giving no platform
+//        let source = """
+//        @Metadata {
+//            @Available(isBeta: true)
+//            @Available(isBeta: true)
+//        }
+//        """
+//        try assertInvalidDirective(source: source)
+    }
+
+    func testInvalidDuplicateDeprecated() throws {
+        throw XCTSkip("FIXME: isDeprecated is unused (https://github.com/apple/swift-docc/issues/441)")
+
+//        func assertInvalidDirective(source: String) throws {
```

Comment:
Do we need to comment these out since we're XCTSkip'ing them?

Reply:
I went ahead and deleted the tests entirely along with the `isBeta`/`isDeprecated` arguments.